### PR TITLE
Update Logging FileHandler Location

### PR DESCRIPTION
**Pull request by wbooth**
update the logging file handler location so logging can be used and imported from anywhere and write to the same directory

### CODE REVIEW

File: app/utility/logger.py [hunk 1]
```diff
@@ -8,7 +8,9 @@ class Logger:
     def __init__(self, name):
         self.name = name
         self.logger = logging.getLogger(name)
-        handler = logging.FileHandler('%s/%s.log' % ('logs', name))
+
+        project_root = Path(__file__).parents[2]
```
Comment (privateducky) [inline app/utility/logger.py]: what problem are you trying to solve? starting the application from a non-root directory?

Comment (wbooth) [reply app/utility/logger.py]: yep, to update the logging file handler location so logging can be used and imported from anywhere and write to the same directory

Comment (privateducky) [reply app/utility/logger.py]: not sure I understand. so logging can occur from anywhere in the application right now.. right? I think I need to see an example of how this adds a benefit.

Comment (wbooth) [reply app/utility/logger.py]: I'll pursue mock , will probably fit better for the use case

**Pull request closed without merging**